packages/block-library/src/term-name/block.json

```
@@ -0,0 +1,68 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"__experimental": true,
+	"name": "core/term-name",
+	"title": "Term Name",
+	"category": "theme",
+	"description": "Displays the name of a taxonomy term.",
+	"textdomain": "default",
+	"usesContext": [ "termId", "taxonomy" ],
+	"attributes": {
+		"textAlign": {
+			"type": "string"
+		},
+		"tagName": {
+			"type": "string",
+			"default": "p"
+		},
```

```suggestion
		"level": {
			"type": "number",
			"default": 0
		},
```

Match the implementation with other blocks.

---

Done in https://github.com/WordPress/gutenberg/pull/72129/commits/af78593cbf83ef4a4b53720723a4bb85bf632ced.